src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/Extensions.cs

```diff
@@ -12,69 +11,33 @@ public static class Extensions
         private const string RegexScriptsBlacklist = @"<\s*s\s*c\s*r\s*i\s*p\s*t\s*[^>]*\s*[^>]*\s*[^>]*>";
         private const string RegexInputsBlacklist = @"<\s*i\s*n\s*p\s*u\s*t\s*[^>]*\s*[^>]*\s*[^>]*>";
         private const string RegexObjectsBlacklist = @"<\s*o\s*b\s*j\s*e\s*c\s*t\s*[^>]*\s*[^>]*\s*[^>]*>";
-
-        public const string WhitelistErrorMessage = "{PropertyName} can't contain invalid characters";
-        public static IRuleBuilderOptions<T, string> MustContainAllowedFreeTextCharacters<T>(
-            this IRuleBuilder<T, string> rule)
-        {
-            return rule.Matches(RegexFreeTextWhitelist)
-                .WithMessage(WhitelistErrorMessage);
-        }
-
-
         // See http://stackoverflow.com/questions/164979/uk-postcode-regex-comprehensive
-        public const string PostcodeRegex = "^(([gG][iI][rR] {0,}0[aA]{2})|((([a-pr-uwyzA-PR-UWYZ][a-hk-yA-HK-Y]?[0-9][0-9]?)|(([a-pr-uwyzA-PR-UWYZ][0-9][a-hjkstuwA-HJKSTUW])|([a-pr-uwyzA-PR-UWYZ][a-hk-yA-HK-Y][0-9][abehmnprv-yABEHMNPRV-Y]))) {0,}[0-9][abd-hjlnp-uw-zABD-HJLNP-UW-Z]{2}))$";
-        public const string InvalidPostCode = "{PropertyName} is invalid";
+        private const string RegexPostcode = "^(([gG][iI][rR] {0,}0[aA]{2})|((([a-pr-uwyzA-PR-UWYZ][a-hk-yA-HK-Y]?[0-9][0-9]?)|(([a-pr-uwyzA-PR-UWYZ][0-9][a-hjkstuwA-HJKSTUW])|([a-pr-uwyzA-PR-UWYZ][a-hk-yA-HK-Y][0-9][abehmnprv-yABEHMNPRV-Y]))) {0,}[0-9][abd-hjlnp-uw-zABD-HJLNP-UW-Z]{2}))$";
```

Comment:
😡 I prefer Regex in suffix, we use suffixes in other cases like Dates, lists

Reply:
Good point, I just made it consistent with the other regex's in the same file. No reason to change it in a different story coming up though.